compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -1926,6 +1932,26 @@ public BSymbol getArithmeticOpsForTypeSets(OperatorKind opKind, BType lhsType, B
         return symTable.notFoundSymbol;
     }
 
+    public BSymbol getUnaryArithmeticOpsForTypeSets(OperatorKind opKind, BType type) {
```

Comment:
These aren't quite "arithmetic operations" right?

Shall we just say

```suggestion
    public BSymbol getUnaryOpsForTypeSets(OperatorKind opKind, BType type) {
```